script/save-cronjob.sh

```diff
@@ -46,5 +49,11 @@ FOLDER=/var/lib/athens/backups/
 TIMESTAMP=$(date +%F-%H-%M)
 FILENAME="${FOLDER}${TIMESTAMP}.edn"
 
+# If java not installed then install it
+
+if [ -z "$(which java)" ]; then
+  apt -y install openjdk-11-jre-headless
```

Comment:
This assumes the linux distro, let's just warn here and `exit 1` so it's clear failure.

Reply:
yeah you are right I was thinking of this being used on servers which are linux but people can also do this while testing locally on other platforms